core/object/object.cpp

```diff
@@ -863,12 +863,15 @@ Variant Object::call_const(const StringName &p_method, const Variant **p_args, i
 
 void Object::notification(int p_notification, bool p_reversed) {
 	if (p_reversed) {
-		if (script_instance) {
-			script_instance->notification(p_notification, p_reversed);
-		}
+		notification_reverse(p_notification);
 	} else {
-		_notificationv(p_notification, p_reversed);
+		notification_forward(p_notification);
 	}
+}
+
+void Object::notification_forward(int p_notification) {
+	constexpr bool p_reversed = false;
```

Comment:
```suggestion
	constexpr bool reversed = false;
```
And just replace, no reason to save a few changed lines